src/mesh/mesh_function.C

```diff
@@ -78,10 +78,18 @@ MeshFunction::MeshFunction (const MeshFunction & mf):
   _vector              (mf._vector),
   _dof_map             (mf._dof_map),
   _system_vars         (mf._system_vars),
-  _out_of_mesh_mode    (false)
+  _out_of_mesh_mode    (mf._out_of_mesh_mode)
 {
-  this->init();
-  this->set_point_locator_tolerance(mf.get_point_locator().get_close_to_point_tol());
+  // Initialize the mf and set the point locator if the
```

Comment:
Reading this comment in Samuel L. Jackson's voice while expanding out the "mf" abbreviation.... :+1: